pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```
@@ -692,6 +701,84 @@ async def finalize_shipment(self, shipment_data: tuple):
+        Errata Tool API. Konflux shipment placeholders are resolved later by promote.
+
+        :param impetus_advisories: Dict of {impetus: advisory_num} from create_et_advisories().
+        :param shipment_data: Tuple of (shipments_by_kind, env, shipment_url) from
+                 prepare_shipment_builds(), or None if no shipment config exists.
+        """
+        rpm_advisory_num = (impetus_advisories or {}).get("rpm")
+        rpm_advisory_id = None
+        if rpm_advisory_num and rpm_advisory_num > 0:
+            try:
+                rpm_advisory_id = get_errata_live_id(rpm_advisory_num)
+            except Exception as ex:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)